### ISSUE TITLE
macOS: development snapshots not showing in updater even though option is checked

### ISSUE THREAD

**Issue by gingerbeardman** [openscad/openscad#3757]
I have the option checked but have not been prompted to install the development snapshot

I just downloaded 2021-05-01 manually.

<img width="636" alt="Screen shot 2021-05-03 at 11 49 20" src="https://user-images.githubusercontent.com/49612/116867806-9cc67d80-ac05-11eb-8d24-1c417fa1e1ce.png">

**Comment by t-paul** [comment]
Yes, we need to find a way to transfer the signing key in a safe way. Currently it's not possible to sign the server file driving the update.

**Comment by gingerbeardman** [comment]
I know this is one of those hard things that may never get done, but for other reasons I am commenting to keep this alive and in the race.
